viite-UI/manual/index.md

```diff
@@ -158,64 +168,79 @@ _Kuva 5.1: Tieosoiteverkon piirtyminen kartalle, kun mittakaavajanassa on 2 km._
+
+_Kuva 5.5: Kokonaisen tieosan valinta._
 
 Kaksoisklikkaus valitsee yhden tielinkin mittaisen osuuden tieosoitteesta. Valittu osuus korostuu kartalla (3), ja sen tiedot tulevat näkyviin karttaikkunan oikean reunan ominaisuustietonäkymään (4).
 
 ![Tieosoitesegmentin valinta](k7.jpg)
 
 _Kuva 5.6: Tielinkin mittaisen osuuden valinta._
 
-5.2 Tieosoitteen ominaisuustiedot
+Ctrl+klik eli monivalinta. Ctrl-nappula pohjassa linkkejä klikkailemalla voit valita vapaasti kaikenlaisia tielinkkejä samaan valintaan.
+Voit poimia esim. tieosoitteistettuja ja tieosoitteistamattomia linkkejä samaan valintaan approksimoidaksesi tulevia tiepituuksia. 
+Monivalituille linkeille ominaisuustiedoissa näytetään vain karkeamman tason tiedot.
+
+![Ctrl+klik; monivalinta, ääriesimerkki](k71.jpg)
+
+_Kuva 5.7: Tielinkkien monivalinta._
+
+5.2 Tieosoitteiden ominaisuustiedot
 --------------------------
 
-Tieosoitteilla on seuraavat ominaisuustiedot:
+Tieosoitteille näytetään seuraavat ominaisuustiedot:
 
-|Ominaisuustieto|Kuvaus|Sovellus muodostaa|
-|---------------|------|----------------|
-|Muokattu viimeksi*|Muokkaajan käyttäjätunnus ja tiedon muokkaushetki.|X|
-|Linkkien lukumäärä|Niiden tielinkkien lukumäärä, joihin valinta  kohdistuu.|X|
-|Geometrian lähde|MML_jäädytetty, MML, Täydentävä tai Suravage.|X|
-|Tienumero|Tieosoiteverkon mukainen tienumero. Lähtöaineistona Tierekisterin tieosoitteet 1.1.2019.||
-|Tieosanumero|Tieosoiteverkon mukainen tieosanumero. Lähtöaineistona Tierekisterin tieosoitteet 1.1.2019.||
-|Ajorata|Tieosoiteverkon mukainen ajoratakoodi. Lähtöaineistona Tierekisterin tieosoitteet 1.1.2019.||
-|Alkuetäisyys**|Tieosoiteverkon etäisyyslukemien avulla laskettu alkuetäisyys. Etäisyyslukeman kohdalla alkuetäisyyden lähtöaineistona on Tierekisterin tieosoitteet 1.1.2019.|X|
-|Loppuetäisyys**|Tieosoiteverkon etäisyyslukemien avulla laskettu loppuetäisyys. Etäisyyslukeman kohdalla loppuetäisyyden lähtöaineistona on Tierekisterin tieosoitteet 1.1.2019.|X|
-|ELY|Väylän ELY-numero.|X|
-|Hallinnollinen luokka***|Saadaan Maanmittauslaitoksen hallinnollinen luokka -tiedoista. Jos valitulla tieosalla on useita hallinnollisia luokkia, ne kerrotaan ominaisuustietotaulussa pilkulla erotettuna.||
-|Jatkuvuus|Tieosoiteverkon mukainen jatkuvuustieto. Lähtöaineistona Tierekisterin tieosoitteet 1.1.2019.|X|
+|Ominaisuustieto   | Kuvaus|
+|------------------|-------|
+|Muokattu viimeksi*| Muokkaajan käyttäjätunnus ja tiedon muokkaushetki.|
+|Linkkien lukumäärä| Niiden tielinkkien lukumäärä, joihin valinta  kohdistuu.|
+|Geometrian lähde  | MML_jäädytetty, MML, Täydentävä tai Suravage.|
+|Kunta             | Geometrialinkin kuntatieto.|
+|Linkin ID         | Näytetään, jos valittuna on yksittäinen linkki.|
+|Geometrian pituus/Geometrioiden yhteenlaskettu pituus| Valittujen tielinkkien yhteispituus. Lasketaan kaikkien linkkien tiedoista.|
+|Tienumero         | Tieosoiteverkon mukainen tienumero. Lähtöaineistona Tierekisterin tieosoitteet (joulukuu 2021).|
```

Comment:
Onko "Lähtöaineistona Tierekisterin tieosoitteet (joulukuu 2021)." tarpeellinen tieto? Parin vuoden päästä uudet Viitteen käyttäjät eivät välttämättä tiedä mikä Tierekisteri on..

Reply:
Hmm, en tiedä, onko tarpeen, ainakaan käyttäjille. Tätä voisi taas kysyä tehokäyttäjiltä.
Itse mietin, onko tuo ehkä tarpeellinen kohta _kehitystiimille_, jos tuo alkulatausaineisto jäätyy kohta Tierekkarin poistumisen myötä? Siinä on viite "checkpointiin"; tällöin ainakin data on mennyt jonkun tsekin läpi.